.golangci.yml

```diff
@@ -16,6 +16,9 @@ linters:
     - scopelint
     - varcheck
     - godox
+    - funlen
+    - whitespace
+    - dogsled
```

Comment:
```suggestion
```

I took a look at [dogsled](https://github.com/alexkohler/dogsled). I think we should actually include this linter and set `n` to a reasonable value. Ideally, these functions return a struct or be refactored entirely. For now, we can set `n` to a reasonable value and add `// nolint: dogsled` where needed.

Reply:
made it three, deafult is two, now it passes